### ISSUE TITLE
Training AUC  Problem

### ISSUE THREAD

**Issue by Yirzzzz** [OUC-VAS/ForensicsAdapter#13]
Hello! First of all, sincere congratulations on your paper FA being accepted to CVPR 2025!

I have downloaded the dataset and successfully run your code on a cloud server with an RTX 4090 GPU. You mentioned in your paper that the best results typically appear in the first half of the initial training cycle, around the 1/4, 1/3, or 1/2 mark. However, when I tried at the 1/3 point, the AUC I obtained on Celeb-DF-v2 was only 72%. Why might this be happening?

**Comment by ccdou** [comment]
You’ve provided too little information. Perhaps you could check whether your training dataset—especially the mask—has been loaded successfully.

**Comment by Yirzzzz** [comment]
> You’ve provided too little information. Perhaps you could check whether your training dataset—especially the mask—has been loaded successfully.

Thanksss!!!! ,it work effectively

<img width="1580" height="119" alt="Image" src="https://github.com/user-attachments/assets/af10e4f3-9a13-4244-af68-4790e6b1c944" />

**Comment by sakisang** [comment]
> > You’ve provided too little information. Perhaps you could check whether your training dataset—especially the mask—has been loaded successfully.
> 
> Thanksss!!!! ,it work effectively
> 
> <img alt="Image" width="1580" height="119" src="https://private-user-images.githubusercontent.com/164160812/481543303-af10e4f3-9a13-4244-af68-4790e6b1c944.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNzQ4NDIsIm5iZiI6MTc1ODE3NDU0MiwicGF0aCI6Ii8xNjQxNjA4MTIvNDgxNTQzMzAzLWFmMTBlNGYzLTlhMTMtNDI0NC1hZjY4LTQ3OTBlNmIxYzk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxOFQwNTQ5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjU4NWMyMzVhNTkzOTM5Y2E5MzJlMjdmOGE3OWMwOGI4MjE3MzExNWRmNzc5NDAxMzc5YTJlMWFmYzM5OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._EUrM5OfIxpqSRXIqWIqgnQMsh2K-AOwZPgmMrf0GgU">

May I ask how you solved this problem?

**Comment by Yirzzzz** [comment]
> > > You’ve provided too little information. Perhaps you could check whether your training dataset—especially the mask—has been loaded successfully.
> > 
> > 
> > Thanksss!!!! ,it work effectively
> > <img alt="Image" width="1580" height="119" src="https://private-user-images.githubusercontent.com/164160812/481543303-af10e4f3-9a13-4244-af68-4790e6b1c944.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNzQ4NDIsIm5iZiI6MTc1ODE3NDU0MiwicGF0aCI6Ii8xNjQxNjA4MTIvNDgxNTQzMzAzLWFmMTBlNGYzLTlhMTMtNDI0NC1hZjY4LTQ3OTBlNmIxYzk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxOFQwNTQ5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjU4NWMyMzVhNTkzOTM5Y2E5MzJlMjdmOGE3OWMwOGI4MjE3MzExNWRmNzc5NDAxMzc5YTJlMWFmYzM5OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._EUrM5OfIxpqSRXIqWIqgnQMsh2K-AOwZPgmMrf0GgU">
> 
> May I ask how you solved this problem?

You could check your mask image paths. I met this problem because my mask path is not provided successfully

**Comment by sakisang** [comment]
> > > > 你提供的信息太少了。也许您可以检查您的训练数据集（尤其是掩码）是否已成功加载。
> > > 
> > > 
> > > 谢谢!!!，它有效地工作
> > > <img alt="Image" width="1580" height="119" src="https://private-user-images.githubusercontent.com/164160812/481543303-af10e4f3-9a13-4244-af68-4790e6b1c944.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNzQ4NDIsIm5iZiI6MTc1ODE3NDU0MiwicGF0aCI6Ii8xNjQxNjA4MTIvNDgxNTQzMzAzLWFmMTBlNGYzLTlhMTMtNDI0NC1hZjY4LTQ3OTBlNmIxYzk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxOFQwNTQ5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjU4NWMyMzVhNTkzOTM5Y2E5MzJlMjdmOGE3OWMwOGI4MjE3MzExNWRmNzc5NDAxMzc5YTJlMWFmYzM5OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._EUrM5OfIxpqSRXIqWIqgnQMsh2K-AOwZPgmMrf0GgU">
> > 
> > 
> > 请问你是如何解决这个问题的？
> 
> 您可以检查蒙版图像路径。我遇到此问题是因为我的掩码路径未成功提供

Thank you, it's been resolved. Also, could you tell me after how many epochs the best result was achieved? Do you mean the author reached the optimum already in the first epoch?

**Comment by Yirzzzz** [comment]
> > > > > 你提供的信息太少了。也许您可以检查您的训练数据集（尤其是掩码）是否已成功加载。
> > > > 
> > > > 
> > > > 谢谢!!!，它有效地工作
> > > > <img alt="Image" width="1580" height="119" src="https://private-user-images.githubusercontent.com/164160812/481543303-af10e4f3-9a13-4244-af68-4790e6b1c944.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNzQ4NDIsIm5iZiI6MTc1ODE3NDU0MiwicGF0aCI6Ii8xNjQxNjA4MTIvNDgxNTQzMzAzLWFmMTBlNGYzLTlhMTMtNDI0NC1hZjY4LTQ3OTBlNmIxYzk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxOFQwNTQ5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjU4NWMyMzVhNTkzOTM5Y2E5MzJlMjdmOGE3OWMwOGI4MjE3MzExNWRmNzc5NDAxMzc5YTJlMWFmYzM5OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._EUrM5OfIxpqSRXIqWIqgnQMsh2K-AOwZPgmMrf0GgU">
> > > 
> > > 
> > > 请问你是如何解决这个问题的？
> > 
> > 
> > 您可以检查蒙版图像路径。我遇到此问题是因为我的掩码路径未成功提供
> 
> Thank you, it's been resolved. Also, could you tell me after how many epochs the best result was achieved? Do you mean the author reached the optimum already in the first epoch?

In my opinion, you could get the best result in the first epoch. it means you don't utilize all your train data in the first epoch. You may randomly get the best result in 1/4, 1/3, and 1/2.

**Comment by sakisang** [comment]
> > > > > > 你提供的信息太少了。也许您可以检查您的训练数据集（尤其是掩码）是否已成功加载。
> > > > > 
> > > > > 
> > > > > 谢谢!!!，它有效地工作
> > > > > <img alt="Image" width="1580" height="119" src="https://private-user-images.githubusercontent.com/164160812/481543303-af10e4f3-9a13-4244-af68-4790e6b1c944.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgxNzQ4NDIsIm5iZiI6MTc1ODE3NDU0MiwicGF0aCI6Ii8xNjQxNjA4MTIvNDgxNTQzMzAzLWFmMTBlNGYzLTlhMTMtNDI0NC1hZjY4LTQ3OTBlNmIxYzk0NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwOTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDkxOFQwNTQ5MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjU4NWMyMzVhNTkzOTM5Y2E5MzJlMjdmOGE3OWMwOGI4MjE3MzExNWRmNzc5NDAxMzc5YTJlMWFmYzM5OTRjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9._EUrM5OfIxpqSRXIqWIqgnQMsh2K-AOwZPgmMrf0GgU">
> > > > 
> > > > 
> > > > 请问你是如何解决这个问题的？
> > > 
> > > 
> > > 您可以检查蒙版图像路径。我遇到此问题是因为我的掩码路径未成功提供
> > 
> > 
> > Thank you, it's been resolved. Also, could you tell me after how many epochs the best result was achieved? Do you mean the author reached the optimum already in the first epoch?
> 
> In my opinion, you could get the best result in the first epoch. it means you don't utilize all your train data in the first epoch. You may randomly get the best result in 1/4, 1/3, and 1/2.
好的感谢，请问每次训练的时候，您的指标是否一样。是否有可复现性。
